src/tests/requirements-test.txt

```diff
@@ -1,10 +1,11 @@
 apache-airflow==2.5.3
```

Comment:
Most of these requirements should be included in `apache-airflow[google,ssh,cncf.kubernetes]==2.5.3` - from what I've looked at only `redshift_connector`, `oracledb`, and `configparser` need to be included separately.